src/test/regress/sql/multi_extension.sql

```diff
@@ -634,11 +634,14 @@ CREATE DATABASE another;
 
 \c another
 CREATE EXTENSION citus;
+\c - - - :worker_1_port
```

Comment:
one another option could be to add `worker_1_port` as a worker in the `another` database. But, this also seems fine.